p384/src/arithmetic/scalar.rs

```diff
@@ -35,7 +35,7 @@ impl ScalarArithmetic for NistP384 {
 /// Scalars are elements in the finite field modulo n.
 #[derive(Clone, Copy, Debug)]
 #[cfg_attr(docsrs, doc(cfg(feature = "arithmetic")))]
-pub struct Scalar(U384);
+pub struct Scalar(pub(crate) U384);
```

Comment:
I hope this is okay.

Reply:
That's fine, although potentially avoidable if all of the constants are given in Montgomery form

Reply:
Didn't need it anymore, so I reverted it.